src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,51 @@
-import React from 'react';
+import React, { FormEvent, useState } from 'react';
+import classNames from 'classnames';
+import { getMovie } from '../../api';
 import './FindMovie.scss';
+import { MovieCard } from '../MovieCard';
+import { MovieData } from '../../types/MovieData';
+
+type Props = {
+  addMovie: (newMovie: MovieData | null) => void
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [movie, setMovie] = useState<MovieData | null>(null);
+  const [query, setQuery] = useState('');
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const handleNewMovie = async (event: FormEvent) => {
+    event.preventDefault();
+    setIsLoading(true);
+
+    const movieFromServer = await getMovie(query);
+
+    if (!movieFromServer.imdbID) {
+      setIsError(true);
+      setMovie(null);
+    }
+
+    if (movieFromServer.imdbID) {
+      setMovie(movieFromServer);
+      setIsError(false);
+    }
```

Comment:
You could use else here 
```suggestion
    if (!movieFromServer.imdbID) {
      setIsError(true);
      setMovie(null);
    } else {
      setMovie(movieFromServer);
      setIsError(false);
    }
```